### PR TITLE
Add "Maven" as required build dependency to "Contributing"

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -34,15 +34,16 @@ Working on your first Pull Request? You can learn how from this free series, [Ho
...
 
 1. Install the required dependencies for your OS:
     1. [Git](https://git-scm.com/downloads)
     1. [GNU make](https://www.gnu.org/software/make/) (probably best to install from your OS package manager such as apt or brew)
     1. [Docker](https://docs.docker.com/get-docker/)
     1. [Go](https://golang.org/doc/install)
     1. [Node.js & npm](https://www.npmjs.com/get-npm)
-    1. *Optional* - [PostgreSQL 11](https://www.postgresql.org/docs/11/tutorial-install.html) (useful for running and debugging locally)
+    2. [Maven](https://maven.apache.org/) to build and test Spark client codes.
+    3. *Optional* - [PostgreSQL 11](https://www.postgresql.org/docs/11/tutorial-install.html) (useful for running and debugging locally)
```
Comment (nopcoder) [inline docs/contributing.md:46]:
```suggestion
    1. [Maven](https://maven.apache.org/) to build and test Spark client codes.
    1. *Optional* - [PostgreSQL 11](https://www.postgresql.org/docs/11/tutorial-install.html) (useful for running and debugging locally)
```

**Comment by arielshaqed** [comment]
@nopcoder PTAL...

**Review by adunuthulan — APPROVED** [review]
Looks good!

**Comment by arielshaqed** [comment]
Thanks!

**Pull request merged**